tests/merch.test.ts

```diff
@@ -0,0 +1,63 @@
+import { ForbiddenError } from 'routing-controllers';
+import { UserAccessType } from '../types';
+import { DatabaseConnection, MerchFactory, UserFactory, PortalState } from './data';
+import { ControllerFactory } from './controllers';
+
+beforeAll(async () => {
+  await DatabaseConnection.connect();
+});
+
+beforeEach(async () => {
+  await DatabaseConnection.clear();
+});
+
+afterAll(async () => {
+  await DatabaseConnection.clear();
+  await DatabaseConnection.close();
+});
+
+describe('collection editing', () => {
+  test('Set collection to archive', async () => {
+    const conn = await DatabaseConnection.get();
+    const itemOpt = MerchFactory.fakeOption();
+    const [item] = MerchFactory.itemsWith({
```

Comment:
nit: rename `itemOpt` -> `itemOption`: we should avoid shorthand for clarity and consistency